src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,43 +1,77 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 
-export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
-
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+type Props = {
+  showTodoId: number;
+  isModalActive: boolean;
+  todo: Todo | null;
+  setIsModalActive: (a: boolean) => void;
+  todoOwner: User | null;
+  modalLoading: boolean;
+};
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
-          </header>
+export const TodoModal: React.FC<Props> = ({
+  showTodoId,
+  isModalActive,
+  todo,
+  setIsModalActive,
+  todoOwner,
+  modalLoading,
+}) => {
+  return (
+    <>
+      {isModalActive && (
```

Comment:
You can do this check in the parent component as you render nothing in this case